presto-main/src/main/java/com/facebook/presto/memory/ClusterMemoryLeakDetector.java

```diff
@@ -65,7 +65,7 @@ void checkForMemoryLeaks(Supplier<List<QueryInfo>> queryInfoSupplier, Map<QueryI
                 .collect(toImmutableMap(Entry::getKey, Entry::getValue));
 
         if (!leakedQueryReservations.isEmpty()) {
-            log.warn("Memory leak detected. The following queries are already finished, " +
+            log.debug("Memory leak detected. The following queries are already finished, " +
```

Comment:
Why? 😕

Reply:
Travis had plenty of these logs causing the log size threshold to be exceeded. (this is like a temp solution to get rid of the travis issue, going forward we need to fix all the leaks).